nav2_system_tests/src/planning/planner_tester.cpp

```diff
@@ -347,57 +316,28 @@ bool PlannerTester::plannerTest(
 
 void PlannerTester::updateRobotPosition(const geometry_msgs::msg::Point & position)
 {
-  geometry_msgs::msg::TransformStamped tf_stamped;
-  tf_stamped.header.frame_id = "map";
-  tf_stamped.header.stamp = now() + rclcpp::Duration(1.0);
-  tf_stamped.child_frame_id = "base_link";
-  tf_stamped.transform.translation.x = position.x;
-  tf_stamped.transform.translation.y = position.y;
-  tf_stamped.transform.rotation.w = 1.0;
-
+  base_transform_->header.stamp = now() + rclcpp::Duration(1.0);
+  base_transform_->transform.translation.x = position.x;
+  base_transform_->transform.translation.y = position.y;
+  base_transform_->transform.rotation.w = 1.0;
   tf2_msgs::msg::TFMessage tf_message;
-  tf_message.transforms.push_back(tf_stamped);
+  tf_message.transforms.push_back(*base_transform_);
   transform_publisher_->publish(tf_message);
```

Comment:
I was seeing that sometimes the transform doesn't get published, that why I added a timer in #1083.

Reply:
Got it, any action here or just mentioning? I looked over that PR and it looks good

Reply:
If you're not seeing the same problem on your tests, then no action.